packages/http-client-csharp/generator/TestProjects/CadlRanch.Tests/Http/Payload/Multipart/MultipartTests.cs

```diff
@@ -219,5 +225,185 @@ private Task MultiBinaryParts(bool hasPicture) => Test(async (host) =>
             var response = await new MultiPartClient(host, null).GetFormDataClient().MultiBinaryPartsAsync(content, content.ContentType, null);
             Assert.AreEqual(204, response.GetRawResponse().Status);
         });
+
+        internal partial class MultiPartFormDataBinaryContent : BinaryContent
```

Comment:
I have to copy this here because despite this is generated, it is internal.
We cannot use internal any more in this structure.

Reply:
Why not? How did it work before? Were we using InternalsVisibleTo?

Reply:
no, we did not.
and since this must be added to the source code's csproj, and that csproj is generated as well, we might not be able to do it.

Reply:
BTW you might notice that in other cases such as the discriminator tests, I changed those test cases to use reflection to do the type checks/property checks because something is actually internal.

Reply:
I also wonder how we expect our customer to use the MFD operations because this class is internal?
We are using this in our test, customers cannot use it therefore does this mean that they need to copy this class into their source code and do the same?